packages/flutter_tools/lib/src/version.dart

```diff
@@ -73,9 +80,15 @@ class FlutterVersion {
 
   String _runGit(String command) => runSync(command.split(' '), workingDirectory: Cache.flutterRoot);
 
+  String _getSdkVersion() {
+    final File versionFile = fs.file('VERSION');
```

Comment:
This will try to find a `VERSION` file in your current working directory - and will throw on the next line if that file doesn't exist.  You should construct an absolute path from a known root location.

Reply:
Ah, so something like this?

```
final File versionFile = fs.file(fs.path.join(getFlutterRoot(), 'VERSION'));
```

Reply:
Yup - or slightly more readable (but one character longer):

```
final File versionFile = fs.directory(getFlutterRoot()).childFile('VERSION');
```